src/main/java/uk/nhs/hee/tis/trainee/forms/dto/LtftFormDto.java

```diff
@@ -74,6 +75,10 @@ public record LtftFormDto(
     CctChangeDto change,
     ReasonsDto reasons,
 
+    @JsonView(ReadOnly.class)
+    @Null(groups = {Create.class, Update.class})
+    EmailValidityType tpdEmailStatus,
```

Comment:
Left as 'tpdEmailStatus' to avoid breaking change for the FE; similarly, 'UNKNOWN' has been used in the enum instead of 'UNVERIFIED' to keep things consistent with current FE listing.